pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -288,7 +297,9 @@ func (l *LibvirtDomainManager) setGuestTime(vmi *v1.VirtualMachineInstance) erro
 
 					switch libvirtError.Code {
 					case libvirt.ERR_AGENT_UNRESPONSIVE:
-						log.Log.Object(vmi).Reason(err).Warning("failed to set time: QEMU agent unresponsive")
+						const unresponsive = "failed to set time: QEMU agent unresponsive"
+						latestErr = fmt.Errorf("%s, %s", unresponsive, err)
+						log.Log.Object(vmi).Reason(err).V(9).Warning(unresponsive)
```

Comment:
I am just thinking: how do we determine the correct log level for a message? I think currently the max in the codebase is `5`. Here we have `9`... If as a user, I wanted to enable *all* the logs, then I would need to dice into the code base. Would it be beneficial to have MaxLogLevel documented?

Reply:
Good point @vasiliy-ul .
We are more or less bound to what is Kubernetes doing imho. Kubernetes uses the whole range 1-9. This area is certainly undocumented. 

NOTE:
See EXT_LOG_VERBOSITY_THRESHOLD in https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-controller/services/template.go#L418 . We are indirectly using more than 5.